actors/miner/tests/util.rs

```diff
@@ -204,8 +261,1024 @@ impl ActorHarness {
+            // unused
+            replace_capacity: false,
+            replace_sector_deadline: 0,
+            replace_sector_partition: 0,
+            replace_sector_number: 0,
+        }
+    }
+
+    fn make_prove_commit_params(&self, sector_no: u64) -> ProveCommitSectorParams {
+        ProveCommitSectorParams { sector_number: sector_no, proof: vec![0u8; 192] }
+    }
+
+    fn pre_commit_sector(
+        &self,
+        rt: &mut MockRuntime,
+        params: PreCommitSectorParams,
+        conf: PreCommitConfig,
+        first: bool,
+    ) -> SectorPreCommitOnChainInfo {
+        rt.set_caller(*ACCOUNT_ACTOR_CODE_ID, self.worker);
+        rt.expect_validate_caller_addr(self.caller_addrs());
+        self.expect_query_network_info(rt);
+
+        if params.deal_ids.len() > 0 {
+            let vdparams = VerifyDealsForActivationParams {
+                sectors: vec![SectorDeals {
+                    sector_expiry: params.expiration,
+                    deal_ids: params.deal_ids.clone(),
+                }],
+            };
+            let vdreturn = VerifyDealsForActivationReturn {
+                sectors: vec![SectorWeights {
+                    deal_space: conf.deal_space.unwrap() as u64,
+                    deal_weight: conf.deal_weight,
+                    verified_deal_weight: conf.verified_deal_weight,
+                }],
+            };
+
+            rt.expect_send(
+                *STORAGE_MARKET_ACTOR_ADDR,
+                MarketMethod::VerifyDealsForActivation as u64,
+                RawBytes::serialize(vdparams).unwrap(),
+                TokenAmount::from(0),
+                RawBytes::serialize(vdreturn).unwrap(),
+                ExitCode::Ok,
+            );
+        }
+        // in the original test the else branch does some redundant checks which we can omit.
+
+        let state = self.get_state(rt);
+        if state.fee_debt > TokenAmount::from(0) {
+            rt.expect_send(
+                *STORAGE_POWER_ACTOR_ADDR,
+                PowerMethod::EnrollCronEvent as u64,
+                RawBytes::default(),
+                state.fee_debt.clone(),
+                RawBytes::default(),
+                ExitCode::Ok,
+            );
+        }
+
+        if first {
+            let dlinfo = new_deadline_info_from_offset_and_epoch(
+                &rt.policy,
+                state.proving_period_start,
+                rt.epoch,
+            );
+            let cron_params = make_deadline_cron_event_params(dlinfo.last());
+            rt.expect_send(
+                *STORAGE_POWER_ACTOR_ADDR,
+                PowerMethod::EnrollCronEvent as u64,
+                RawBytes::serialize(cron_params).unwrap(),
+                TokenAmount::from(0),
+                RawBytes::default(),
+                ExitCode::Ok,
+            );
+        }
+
+        rt.call::<Actor>(
+            Method::PreCommitSector as u64,
+            &RawBytes::serialize(params.clone()).unwrap(),
+        )
+        .unwrap();
```

Comment:
nit: you could assert that this is the expected `RawBytes::default()` here

Reply:
sure.

Reply:
I am adding an `expect_empty` util method to the test utils so that we can use that consistently.